doc/introduction.rst

```diff
@@ -0,0 +1,82 @@
+problems for **scientific machine learning (SciML)** models that combine machine learning
+and mechanistic ordinary differential equation (ODE) models.
+
+Highlights
+----------
+
+Extending the `PEtab format <https://petab.readthedocs.io/>`_ for mechanistic models,
+PEtab SciML provides an accessible, reproducible way to specify SciML training problems
+across diverse scenarios, in a format directly importable by downstream tools. The main
+aspects enabling this are:
+
+- **Flexible hybridization.** Machine learning (ML) and ODE models can be combined in three
+  ways: (1) ML within the ODE dynamics (includes **Neural ODEs**), (2) ML in the
+  observable/measurement model linking simulations to data, and (3) ML upstream of the ODE,
+  mapping high-dimensional inputs (e.g., images) to ODE model parameters.
+- **Import across ecosystems.** PEtab SciML problems can be imported into state-of-the-art
+  toolboxes for dynamic-model training in Julia
+  (`PEtab.jl <https://github.com/sebapersson/PEtab.jl>`_) and Python/JAX
+  (`AMICI <https://github.com/AMICI-dev/AMICI>`_).
+- **Diverse model types.** All model features of the
+  `PEtab format <https://petab.readthedocs.io/>`_ are supported, like models with partial
+  observability, multiple simulation conditions, diverse noise models, and events/callbacks.
+- **Efficient training strategies.** With minimal user input, PEtab SciML problems can be
+  rewritten at the PEtab abstraction level to be compatible with training strategies such as
+  multiple shooting, curriculum learning, and regularization (e.g., of ML outputs).
+- **Thoroughly tested.** An extensive test suite ensures importers produce correct,
+  consistent output.
+- **Linting and helpers.** The PEtab SciML Python library provides a linter and utility
+  functions for creating common problem types (e.g., Neural ODEs) and transformations (e.g.,
+  rewriting a PEtab problem for multiple-shooting training).
+
+Installation
+------------
+
+The PEtab SciML Python3 helper library can be installed with:
+
+.. code-block:: bash
+
+   pip install petab-sciml
+
+
+How to read the documentation
+-----------------------------
+
+If you are new to PEtab SciML, start with the Getting Started tutorial. It is a
+prerequisite for the How-to guides, which cover different model scenarios (e.g., Neural
+ODEs, ML model upstream of the ODE). For a complete description of all options when
+defining a SciML problem, see the Format specification.
+
+Why a SciML data format?
+------------------------
+
+There are several technical challenges with training SciML models. First, coding a correct
+and performant loss/objective function is non-trivial, especially for more complex models
+like those events or partial observability. Second, efficient training requires gradients
+computed via automatic differentiation (AD). Yet writing fast, AD-compatible code in
+frameworks like JAX, PyTorch, or the Julia SciML ecosystem can be challenging. Similarly,
```

Comment:
```suggestion
computed via automatic differentiation (AD). Although frameworks like JAX,
PyTorch and the Julia SciML ecosystem support AD-compatible code, it can still
be challenging to write performant code. Similarly,
```